pydotorg/urls.py

```diff
@@ -33,7 +33,7 @@
     url(r'^downloads/', include('downloads.urls', namespace='download')),
     url(r'^doc/$', TemplateView.as_view(template_name="python/documentation.html"), name='documentation'),
     #url(r'^community/$', TemplateView.as_view(template_name="python/community.html"), name='community'),
-    url(r'^blog/$', TemplateView.as_view(template_name="python/blog.html"), name='blog'),
+    url(r'^blog/$', RedirectView.as_view(url=settings.PYTHON_BLOG_URL)),
```

Comment:
Wait, on second thought, I think this should be redirected to `/blogs/` instead of blogs.python.org. `/blog/` was probably used as a showcase during development.